examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    data_acquired=[[None], [None], [None]]
</pre><p>this is still super cryptic. you need to explain what will be the elements of this.</p><p> </p><p>Also, why are they 3 long? Is it because you try it on 3 images? Because that seems very inflexible, then I would propose to make it one variable called</p><p> </p><p>acquired_image = [None]</p><p>computed_coordinates = [None] </p><p> </p><p>or something which changes every iteration. The 3 seems very arbitrary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>